types/ui-grid/index.d.ts

```diff
@@ -3799,7 +3801,7 @@ declare namespace uiGrid {
          * If suppressRemoveSort is also set, the unsorted state will be skipped even if it is listed here. Each direction may
          * not appear in the list more than once (e.g. [ASC, DESC, DESC] is not allowed), and the list may not be empty.*
          */
-        sortDirectionCycle?: Array<IUiGridConstants>;
+        sortDirectionCycle?: Array<string>;
```

Comment:
Not every string would be suitable here.  I would rather see a union type of string literals or a type alias of sort directions.  Otherwise it wouldn't complain if you put in something like `'DES'`.